src/item_factory.cpp

```diff
@@ -1422,6 +1444,8 @@ void Item_factory::load_basic_info(JsonObject &jo, itype *new_item_template)
         new_item_template->category = get_category( jo.get_string( "category" ) );
     }
 
+    assign( jo, "faults", new_item_template->faults );
```

Comment:
YAGNI, in other words make it as specific as possible when added. That reminds me, thete's still a bunch of gun specific data in the base itype.